src/pages/components/text-input/usage.mdx

```diff
@@ -233,6 +233,8 @@ character counter.
    the field is optional or required for the user to complete.
 6. **Character counter** (optional): Indicate the number of characters being
    entered and the total number of characters allowed.
+7. **Word counter** (optional): Indicate the number of words being entered and
```

Comment:
![image](https://github.com/carbon-design-system/carbon-website/assets/148799271/a4a469d5-ab62-46b8-be66-e23fba359224)
As there is no '7.' number in the image it could be a little misleading, alternatively what we can do is something like this : 

6. ""Counter**(optional)
     - **Character counter**: Indicate the number of characters being entered and the total number of characters allowed.
     - **Word counter**: Indicate the number of words being entered and the total number of words allowed.

----------

<img width="543" alt="image" src="https://github.com/carbon-design-system/carbon-website/assets/148799271/886c1aa8-5da5-4d94-a4d2-6d7f661fd363">


There is a small typho (typographical error) in "Resize hanle" --> "Resize handle"